VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoPrescriptions.tsx

```diff
@@ -54,10 +54,10 @@ function PrescriptionHistoryNoPrescriptions() {
   return (
     <VAScrollView>
       <Box {...alertWrapperProps}>
-        <AlertBox
-          border={'informational'}
-          title={t('prescriptions.notFound.title')}
-          titleA11yLabel={a11yLabelVA(t('prescriptions.notFound.title'))}>
+        <AlertWithHaptics
+          variant="info"
+          header={t('prescriptions.notFound.title')}
+          headerA11yLabel={a11yLabelVA(t('prescriptions.notFound.title'))}>
           <TextView
```

Comment:
Could potentially be partially simplified moving this to the `description` prop to AlertWithHaptics.